src/device/usbd.c

```diff
@@ -119,7 +119,7 @@ tu_static usbd_class_driver_t const _usbd_driver[] =
     .open             = hidd_open,
     .control_xfer_cb  = hidd_control_xfer_cb,
     .xfer_cb          = hidd_xfer_cb,
-    .sof              = NULL
+    .sof              = hidd_sof_isr
```

Comment:
The assigned ISR function at this place gets called here: https://github.com/hathach/tinyusb/blob/cfe8ca3fa51061bbe7e519b72236f9aabae13e59/src/device/usbd.c#L1132